.github/workflows/release_aur.yml

```diff
@@ -0,0 +1,78 @@
+name: Update AUR
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-aur:
+    runs-on: ubuntu-latest
+
+    container:
+      image: archlinux
+
+    steps:
+      - name: Install Dependencies
+        run: |
+          pacman -Syu --noconfirm wget jq
+
+      - name: Download Package
+        run: |
+          set -x
+          latest_version=$(curl -s https://api.github.com/repos/Heroic-Games-Launcher/HeroicGamesLauncher/releases/latest | jq -r .tag_name)
+          version_without_v=$(echo $latest_version | cut -c 2-)
+          wget https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/download/${latest_version}/heroic-${version_without_v}.pacman -O heroic.pacman
+          set +x
+        if: ${{ success() }}
+
+      - name: Update PKGBUILD
+        run: |
+          set -x
+          latest_version=$(curl -s https://api.github.com/repos/Heroic-Games-Launcher/HeroicGamesLauncher/releases/latest | jq -r .tag_name)
+          pkgver=${latest_version#v}
+          pkgrel=1
+          url="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher"
+          _filename=heroic-${pkgver}.pacman
+          source=("$url/releases/download/$latest_version/$_filename")
+          md5sums=($(md5sum heroic.pacman | cut -d' ' -f1))
+
+          cat <<EOF > PKGBUILD
+          # Maintainer: flaviofearn <flavioislima@gmail.com>
+
+          pkgname=heroic-games-launcher-bin
+          pkgver=$pkgver
+          pkgrel=$pkgrel
+          pkgdesc="An open source Web3 gaming platform."
```

Comment:
That doesn't look right

Reply:
good catch, copy pasted from hyperplay 🤦🏽

Reply:
fixed